### DISCUSSION TITLE
[FEATURE] Decoy Vault

### DISCUSSION THREAD

**Discussion by knightian** [dani-garcia/vaultwarden#2488] [Ideas]
I had a thought about a useful feature.

What if Vaultwarden could detect a wrong password and use it to supply a decoy vault? So you have an account (email) that has it's normal master password, but you have also another master password attached to the same account (email) that if you supply under coercion it unlocks a decoy vault full of decoy password values.

**Comment by BlackDex** [comment]
Sounds like a truecrypt feature.
But that isn't something which can be done by this project it self. Since the clients need to support that too.

Also, if you were coerced, they would probably check the results first and keep trying to get it out of you.

**Reply by knightian** [reply]
Yea spot on truecrypt was my inspiration. Hmm yes I suppose given that the client can be in a locked state using a pin to access etc it would be not a great way to do it without modifying the client
